src/sentry/models/group.py

```diff
@@ -45,6 +54,28 @@ class GroupStatus(object):
 class GroupManager(BaseManager):
     use_for_related_fields = True
 
+    def by_qualified_short_id(self, org, short_id):
```

Comment:
Returning `None` out of this function rather than raising `Group.DoesNotExist` if there is no match doesn't seem like idiomatic Django if this is supposed to match the semantics of `get`.